src/ostorlab/runtimes/local/app/types.py

```diff
@@ -0,0 +1,56 @@
+import graphene
+
+
+class ScanOrderByEnum(graphene.Enum):
+    """Enum for the elements to order a scan by."""
+
+    ScanId = 1
+    Title = 2
+    CreatedTime = 3
+    Progress = 4
+
+
+class ScanType(graphene.ObjectType):
+    """Graphene object type for a scan."""
+
+    id = graphene.Int()
+    title = graphene.String()
+    asset = graphene.String()
+    progress = graphene.String()
+    created_time = graphene.DateTime()
```

Comment:
am not sure if [graphene-sqlalchemy](https://github.com/graphql-python/graphene-sqlalchemy) is worth trying here..